wrap.go

```diff
@@ -136,6 +136,37 @@ func As[T Steper](s Steper) []T {
 	}
 }
 
+// IsStep reports whether there is any step matches target step.
+func IsStep(step, target Steper) bool {
+	if step == target {
+		return true
+	}
+	if step == nil || target == nil {
+		return false
+	}
+	for {
```

Comment:
What is this `for` for?

Reply:
I copy this pattern from `errors.Is`, pls also check the implementation of `flow.Is`